src/commands/domains/link.js

```diff
@@ -0,0 +1,65 @@
+async function action (params) {
+  let c = require('picocolors')
+  let client = require('@begin/api')
+  let { config, domain, appID, env } = params
+  let { access_token: token, stagingAPI: _staging } = config
+
+  if (!domain || (typeof domain === 'string' && domain.length === 0))
+    return Error('Please specify a domain with --domain')
```

Comment:
You might want to create a file like `src/command/env/errors.js` so that we can re-use the error messages and also make translation easier.

Also, if possible do all the error checks and return an array of errors so the user doesn't have to run the command multiple times to figure out what they did wrong.